util/openocd/target/lowrisc-earlgrey.cfg

```diff
@@ -21,6 +21,8 @@ set _TARGETNAME $_CHIPNAME.tap
 target create $_TARGETNAME.0 riscv -chain-position $_TARGETNAME -rtos hwthread
 
 # Configure work area in on-chip SRAM
+#
+# TODO(lowRISC/opentitan#13968) Use a region that won't clobber SRAM programs?
 $_TARGETNAME.0 configure -work-area-phys 0x80000000 -work-area-size 1000 -work-area-backup 0
```

Comment:
SRAM programs are in the main SRAM (origin: 0x10000000, length: 0x20000). Is this still a concern?

Reply:
Oh! I guess that's fine then. I'll drop the TODO.